src/nonlin_svm_model.jl

```diff
@@ -0,0 +1,98 @@
+export svm_train_model, svm_test_model, svm_model
+
+function tan_data_train()
+  #load data
+  A, b = MLDatasets.MNIST(split=:train)[:]
```

Comment:
I'm guessing the problem is there. Don't you need a certain version of MNIST for this syntax to work? Otherwise, Julia thinks you're "calling" the *module* MNIST.

See the error here: https://github.com/JuliaSmoothOptimizers/RegularizedProblems.jl/runs/8181014180?check_suite_focus=true#step:6:417

Click on any of the failed "checks" to see the full log and the error message.

Reply:
The tests are using MLDatasets 0.5.15: https://github.com/JuliaSmoothOptimizers/RegularizedProblems.jl/runs/8181014180?check_suite_focus=true#step:6:40

Check that that's the right version.

Reply:
They should be using 0.7.4